app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  has_many :likes
+  has_many :comments
+  belongs_to :user, foreign_key: true
+  after_save :update_posts_counter
```

Comment:
Good job using callbacks! :+1:

Reply:
Learning from masters, thank you @Nemwel-Boniface .